indigo_api/templates/indigo_api/akn/coverpage_act.html

```diff
@@ -55,27 +55,124 @@ <h2>{{ subt }}</h2>
+                      {% else %}
+                        {% blocktrans %}by <a href="{{ resolver_uri }}">{{ title }}</a>{% endblocktrans %}
+                      {% endif %}
+                    {% endwith %}
+                  {% endif %}
+                {% endwith %}
+              </li>
+          {% endfor %}
+          {% if document.work.uncommenced_provisions %}
+            {% commenced_provisions_description document commencement uncommenced=True as provisions_description %}
+            <li class="commencement-date-uncommenced">
+              {% blocktrans %}{{ provisions_description }}: not yet commenced{% endblocktrans %}
+            </li>
+          {% endif %}
+        {% else %}
+          <table class="commencements-table">
+            <thead>
+              <tr>
+                <th>{% blocktrans %}Provisions{% endblocktrans %}</th>
+                <th>{% blocktrans %}Status{% endblocktrans %}</th>
+              </tr>
+            </thead>
+            <tbody>
+              {% for commencement in commencements %}
+                {% commenced_provisions_description document commencement as provisions_description %}
+                <tr>
+                  <td>
+                    {% if provisions_description %}
+                      {{ provisions_description }}
+                    {% else %}
+                      <i>{% blocktrans %}Unknown provisions{% endblocktrans %}</i>
+                    {% endif %}
+                  </td>
+                  <td>
+                    {% blocktrans with date=commencement.date|date:"j E Y" %}commenced on {{ date }}{% endblocktrans %}
+                    {% with commencement.commencing_work as commencing_work %}
+                      {% if commencing_work %}
+                        {% with numbered_title=commencing_work.numbered_title title=commencing_work.title %}
+                          {% work_resolver_url commencing_work as resolver_uri %}
+                          {% if numbered_title %}
+                            {% blocktrans %}by <a href="{{ resolver_uri }}">{{ numbered_title }}</a>{% endblocktrans %}
+                          {% else %}
+                            {% blocktrans %}by <a href="{{ resolver_uri }}">{{ title }}</a>{% endblocktrans %}
+                          {% endif %}
+                        {% endwith %}
+                      {% endif %}
+                    {% endwith %}
+                  </td>
+                </tr>
+              {% endfor %}
+              {% if document.work.uncommenced_provisions %}
+                <tr>
+                  <td style="font-weight: bold">
+                    {% commenced_provisions_description document commencement uncommenced=True as provisions_description %}
+                    {{ provisions_description }}
+                  </td>
+                  <td style="font-weight: bold">{% blocktrans %}Not yet commenced{% endblocktrans %}</td>
```

Comment:
Lowercase "N", to match the lowercase "c" for the "commenced by" version?

Reply:
absolutely, thanks